packetbeat/packetbeat.yml

```diff
@@ -24,132 +24,131 @@ packetbeat.flows:
-
-    # Uncomment the following to hide certain parameters in URL or forms attached
-    # to HTTP requests. The names of the parameters are case insensitive.
-    # The value of the parameters will be replaced with the 'xxxxx' string.
-    # This is generally useful for avoiding storing user passwords or other
-    # sensitive information.
-    # Only query parameters and top level form parameters are replaced.
-    # hide_keywords: ['pass', 'password', 'passwd']
-
-  memcache:
-    # Configure the ports where to listen for memcache traffic. You can disable
-    # the Memcache protocol by commenting out the list of ports.
-    ports: [11211]
-
-    # Uncomment the parseunknown option to force the memcache text protocol parser
-    # to accept unknown commands.
-    # Note: All unknown commands MUST not contain any data parts!
-    # Default: false
-    # parseunknown: true
-
-    # Update the maxvalue option to store the values - base64 encoded - in the
-    # json output.
-    # possible values:
-    #    maxvalue: -1  # store all values (text based protocol multi-get)
-    #    maxvalue: 0   # store no values at all
-    #    maxvalue: N   # store up to N values
-    # Default: 0
-    # maxvalues: -1
-
-    # Use maxbytespervalue to limit the number of bytes to be copied per value element.
-    # Note: Values will be base64 encoded, so actual size in json document
-    #       will be 4 times maxbytespervalue.
-    # Default: unlimited
-    # maxbytespervalue: 100
-
-    # UDP transaction timeout in milliseconds.
-    # Note: Quiet messages in UDP binary protocol will get response only in error case.
-    #       The memcached analyzer will wait for udptransactiontimeout milliseconds
-    #       before publishing quiet messages. Non quiet messages or quiet requests with
-    #       error response will not have to wait for the timeout.
-    # Default: 200
-    # udptransactiontimeout: 1000
-
-  mysql:
-    # Configure the ports where to listen for MySQL traffic. You can disable
-    # the MySQL protocol by commenting out the list of ports.
-    ports: [3306]
-
-  pgsql:
-    # Configure the ports where to listen for Pgsql traffic. You can disable
-    # the Pgsql protocol by commenting out the list of ports.
-    ports: [5432]
-
-  redis:
-    # Configure the ports where to listen for Redis traffic. You can disable
-    # the Redis protocol by commenting out the list of ports.
-    ports: [6379]
-
-  thrift:
-    # Configure the ports where to listen for Thrift-RPC traffic. You can disable
-    # the Thrift-RPC protocol by commenting out the list of ports.
-    ports: [9090]
-
-  mongodb:
-    # Configure the ports where to listen for MongoDB traffic. You can disable
-    # the MongoDB protocol by commenting out the list of ports.
-    ports: [27017]
-
-  nfs:
-    # Configure the ports where to listen for NFS traffic. You can disable
-    # the NFS protocol by commenting out the list of ports.
-    ports: [2049]
+packetbeat.protocols.icmp:
```

Comment:
I moved them one more level up. I think it's easier to read the configuration file like this. I think it's similar with what ES has now.